client/stylesheets/_toCat.scss

```diff
@@ -5184,16 +5184,66 @@ On click the images will be displayed at normal size to complete the effect
 
 .headerSearchRadiusLogo {
     margin-right: 30px;
-    font-weight: 700;
+    font-weight: 400;
 }
 
 .headerSearchRadiusLogo a {
 	font-size: 1.3rem !important;
 }
 
+.headerSearchNavLogo {
+	margin-left: 16px;
+	margin-top: 3px;
+}
+
 /* End */
 
+/* Nav Search */
 
+.headerSearchNavField {
+	color: #404040;
+    font-weight: 400;
+    font-size: 0.9rem !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces